src/main/java/com/worksap/nlp/sudachi/SudachiCommandLine.java

```diff
@@ -87,25 +92,24 @@ static void run(Tokenizer tokenizer, Tokenizer.SplitMode mode, InputStream input
         }
     }
 
-    static MorphemeFormatterPlugin getFormatter(String path, String jsonString, boolean mergeSettings,
-            boolean isWordSegmentation, boolean isLineBreakAtEosInWordSegmentation) throws IOException {
-        Settings settings = JapaneseDictionary.buildSettings(path, jsonString, mergeSettings);
-        List<MorphemeFormatterPlugin> formatters = settings.getPluginList("formatterPlugin");
-        if (formatters.isEmpty()) {
-            throw new IllegalArgumentException("no morpheme formatter");
-        }
+    static MorphemeFormatterPlugin makeFormatter(boolean isWordSegmentation, boolean isLineBreakAtEosInWordSegmentation)
+            throws IOException {
         MorphemeFormatterPlugin formatter;
+        Settings emptySettings = Settings.parse("{}", SettingsAnchor.none());
         if (isWordSegmentation) {
+            formatter = new WordSegmentationFormatter();
+            formatter.setSettings(emptySettings);
+            formatter.setUp();
             if (isLineBreakAtEosInWordSegmentation) {
-                formatter = formatters.get(1);
+                formatter.setEosString("\n");
             } else {
-                formatter = formatters.get(2);
+                formatter.setEosString(" ");
             }
         } else {
-            formatter = formatters.get(0);
+            formatter = new SimpleMorphemeFormatter();
```

Comment:
Since SimpleMorphemeFormatter is used as the definitive formatter, other formatters cannot be used.
https://github.com/WorksApplications/chikkar#sudachi-%E3%82%B3%E3%83%9E%E3%83%B3%E3%83%89%E3%83%A9%E3%82%A4%E3%83%B3%E3%83%84%E3%83%BC%E3%83%AB%E3%81%A7%E3%81%AE%E5%88%A9%E7%94%A8-using-chikkar-with-the-sudachi-cli

Reply:
Mixing analysis config and morpheme formatting does not seem good for me.

Proposal: introduce `--format` flag which will accept some basic formatter types or Java class name, which will be instantiated to do the formatting.

Reply:
OK. I don't want Chikkar to stop working, so please do something about it.

Reply:
Added `--format` parameter which now accepts full class name of the formatter class. Probably it can be extended to support short aliases for current formatters as well.